spec/requests/api/v1/sessions/create_spec.rb

```diff
@@ -7,6 +7,7 @@
 
   describe 'POST create' do
     let(:password)              { '12345678' }
+    let(:user_password)         { '12345678' }
```

Comment:
Is it possible to keep only one variable for the same value/purpose? :password and :user_password seem to be the same.

Reply:
That's because in context of 'when credentials don't match', needs to update params password. If only one variable is used, it also changes user password and test fails. Maybe you have some suggestion?